cognite/client/data_classes/aggregates.py

```diff
@@ -0,0 +1,9 @@
+from cognite.client.data_classes._base import CognitePropertyClassUtil
+
+# GenClass: DataSet
```

Comment:
This means the DataSet class will be generated from the openapi spec here and overwrite this implementation next time `generate_from_spec.py` is run, which I guess you don't want.

Reply:
oops, thx for the catch!